src/Installers/Windows/Common/dotnethome_x64.wxs

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?ifndef Platform?>
+    <?define Platform = "$(sys.BUILDARCH)"?>
+  <?endif?>
+
+  <!-- InstallerArchitecture matches the expected values for PROCESSOR_ARCHITECTURE
+       https://docs.microsoft.com/en-us/windows/win32/winprog64/wow64-implementation-details  -->
+  <?if $(var.Platform)~=x86?>
+    <?define InstallerArchitecture="X86"?>
+  <?elseif $(var.Platform)~=x64?>
+    <?define InstallerArchitecture="AMD64"?>
+  <?elseif $(var.Platform)~=arm64?>
+    <?define InstallerArchitecture="ARM64"?>
+  <?else?>
+    <?error Unknown platform, $(var.Platform) ?>?
+  <?endif?>
+
+  <Fragment>
+    <!-- Identify when installing in emulation as when PROCESSOR_ARCHITECTURE does not match the installer architecture
+          https://docs.microsoft.com/en-us/windows/win32/winprog64/wow64-implementation-details -->
+    <SetProperty Action="Set_NON_NATIVE_ARCHITECTURE" Id="NON_NATIVE_ARCHITECTURE" Value="true" Before="CostFinalize">
+      NOT %PROCESSOR_ARCHITECTURE="$(var.InstallerArchitecture)"
```

Comment:
Should this be a case-insensitive comparison?

Reply:
The values are documented explicitly as the strings defined previously in this file, case and all.   I did the mapping intentionally.

Also this goes away once we have the wix feature where will compare against numeric values for image file machine.